packages/store-sdk/src/analytics/events.ts

```diff
@@ -0,0 +1,185 @@
+export interface PageViewData {
+  accountName: string
+  pageTitle: string
+  pageUrl: string
+  referrer: string
+  pageType: 'home' | 'fullText' | 'pdp' | 'plp'
+}
+
+export interface UserData extends PageViewData {
+  firstName?: string
+  lastName?: string
+  document?: string
+  id?: string
+  email?: string
+  phone?: string
+  isAuthenticated: boolean
+}
+
+export interface CartIdData extends PageViewData {
+  cartId: string
+}
+
+export type ProductPageInfoData = PageViewData
+
+export interface CategoryViewData extends PageViewData {
```

Comment:
can we remove CategoryViewData?

Reply:
All these events will be restructured in a future PR.